packages/asset-uploader/package.json

```diff
@@ -11,6 +11,7 @@
     "checks": "pnpm typecheck && pnpm lint && pnpm test",
     "dev": "build-package --watch",
     "build": "build-package",
+    "dts": "tsc --emitDeclarationOnly --declaration --declarationDir lib/types",
```

Comment:
Does it still require "include": ["src"] in tsconfig? Because of it .ts files in the root of packages are not checked.

Reply:
I wasn't exactly sure why you were using `include: ['src']` in the first place so I left it alone. If you don't need it then we could likely remove it from `tsconfig.json` **and** pass it to the args list here (we want `./lib/types` output to reflect `./src` and not `./`, so we need it for the dts emit itself)

Reply:
I would move this particular work to another PR though to keep the changes incremental and smaller